src/Sylius/Behat/Context/Api/Admin/ManagingProductsContext.php

```diff
@@ -355,6 +432,16 @@ public function thisProductShouldHaveOption(ProductInterface $product, ProductOp
         );
     }
 
+    /**
+     * @Then the first product on the list should have :field :value
+     */
+    public function theFirstProductOnTheListShouldHave(string $field, string $value): void
+    {
+        $products = $this->responseChecker->getCollection($this->client->index());
```

Comment:
Check out [this comment](https://github.com/Sylius/Sylius/pull/11243#discussion_r393573401), I think we should use `getLastReponse()` whenever the name of the step clearly says we have an index as a current response (product **on the list**)